InvenTree/plugin/api.py

```diff
@@ -164,18 +197,10 @@ def get_object(self):
         plugin_slug = self.kwargs['plugin']
         key = self.kwargs['key']
 
-        # Check that the 'plugin' specified is valid!
-        if not PluginConfig.objects.filter(key=plugin_slug).exists():
-            raise NotFound(detail=f"Plugin '{plugin_slug}' not installed")
-
-        # Get the list of settings available for the specified plugin
-        plugin = registry.get_plugin(plugin_slug)
-
-        if plugin is None:
-            # This only occurs if the plugin mechanism broke
-            raise NotFound(detail=f"Plugin '{plugin_slug}' not found")  # pragma: no cover
+        # Look up plugin
+        plugin = check_plugin(plugin_slug)
 
-        settings = getattr(plugin, 'SETTINGS', {})
+        settings = getattr(plugin, 'settings', {})
```

Comment:
@matmair are you sure this won't break existing code?

Reply:
@SchrodingersGat It might but the current behavior is wrong and inconsistent.
SETTINGS should be a static constant following python naming conventions while settings is a variable. In the plugin-loading process settings is set from SETTINGS but could be changed later (either directly or by modifying the loading method) to dynamically change the provided settings. This functionality is present in multiple mixins and also documented in the docstrings. So the current usage of SETTINGS is just wrong.